environment_linux.yml

```diff
@@ -33,12 +33,11 @@ dependencies:
   - cudatoolkit=10.2
   - opencv=4.5.0 # preferring conda-forge's distribution as it provides the latest distribution
   - pytorch
+  - embree
```

Comment:
do we need embree and trimesh for this PR?

Reply:
Preferably

> Is the figure you posted
> 
> ```
> Current results on 2011205_opnav_022 segment of the Dawn mission to 4 Vesta.
> ```
> 
> from the deep front end?

Yes it is

Reply:
I can comment it out since it's not really being used yet, but could I leave it commented out so I don't have to rewrite it later?

Reply:
so i think we should only check in these dependencies after evaluating the specific use case and tradeoffs with other libraries.

do they have python wheels in conda for all platforms for python 3.8?

Reply:
We don't _need_ embree with trimesh, but conducting ray tracing without it is orders of magnitude slower (a couple minutes as opposed to hundredths of a second). 

I've looked into this a lot actually, as it's kind of the backbone of my data generation scheme, and `trimesh` with `embree` are definitely the best for performing ray tracing in python.

`trimesh` by itself is also a really great library on it's own for any tasks involving meshes. It also has a lot of functionality with `o3d` for plotting. 

I can of course forego the ray tracing and ground truth mesh and just use the depth maps if you'd prefer, but it's useful to have the actual ground truth mesh for plotting and evaluation purposes.

Let me know what you think.

Reply:
No great downside, right?

Reply:
Cool, can we just double check to make sure that wheels are available for both packages in conda for python 3.8 for both mac and linux?

Reply:
@dellaert Only upsides :)

However, I've removed all the dependencies on `trimesh` and `embree` as @johnwlambert suggested, as the ground truth mesh isn't being utilized yet in this PR.